src/main/java/com/smartnews/jpa_entity_generator/util/TypeConverter.java

```diff
@@ -18,8 +18,8 @@ public static String toJavaType(int typeCode) {
                 return "Long";
             // case Types.BINARY:
             case Types.BIT:
-                // return "Boolean";
-                return "boolean";
+                // my sql
```

Comment:
This comment is not necessary as with my comment `return "Boolean";` above 😅

Reply:
cleaned